.github/workflows/tests.yml

```diff
@@ -91,12 +88,12 @@ jobs:
           pip install nox
           nox -s compile
       - name: Run tests
-        run: pytest -m "not a11y" --color=yes --cov pydata_sphinx_theme --cov-branch --cov-report xml:cov.xml --cov-report= --cov-fail-under ${{ env.COVERAGE_THRESHOLD }}
+        run: pytest -m "not a11y" --color=yes --cov --cov-report=xml
```

Comment:
this doesn't look right to me:

- the second `--cov-report=` with no argument is supposed to suppress the coverage output in the terminal output (which we don't need because we're generating XML and then viewing it through CodeCov website)
- I think we still want `--branch` (can be `branch = true` under `tool.coverage.run` in the TOML file if you prefer)

Reply:
I prefer to limit the number of parameter in the github actions so that we can rely exclusively on the nox session in the future. I'll add them to the TOML file

Reply:
I checked the difference between the output of the current main and this branch and I don't see the difference in term of terminal output: 
<img width="961" alt="image" src="https://github.com/pydata/pydata-sphinx-theme/assets/12596392/39144be1-55ce-47a9-bc2a-25e2bdecc4ed">

@drammock, are you sure the second `--cov-report` is really changing anything in the terminal output? (left is the current PR, right is the current main)

Reply:
Weird, that's what is supposed to happen according to the docs but I agree that it's definitely not looking different. 🤷🏻